static/js/pages/applications/ApplicationDashboardPage.js

```diff
@@ -426,12 +427,27 @@ export class ApplicationDashboardPage extends React.Component<Props, State> {
       />
     )
 
+    const isNovoEdCourse = !!applicationDetail.bootcamp_run.novoed_course_stub
+    const novoEdEnrolled =
+      isNovoEdCourse &&
+      !!SETTINGS.novoed_login_url &&
+      !!applicationDetail.enrollment &&
+      !!applicationDetail.enrollment.novoed_sync_date
+    const bootcampStartRow = isNovoEdCourse ? (
+      <BootcampStart
+        ready={novoEdEnrolled}
+        fulfilled={novoEdEnrolled}
```

Comment:
Looks like `ready` and `fulfilled` will always be the same, so one of them could be removed as a prop.  Or is there a future condition/use-case where that might not be true?

Reply:
Yup they'll always be the same (for now), but the rest of the detail sections follow a pattern where they receive those two props and they can be different values. To me it makes sense to keep the props consistent even if at the moment they will always be given the same value